index.mjs

```diff
@@ -8,30 +8,30 @@ export function ready(cb) {
   wrapped.ready().then(cb);
 }
 
-export function verifySignatures(msgs, cb) {
-  wrapped.verifySignatures(msgs).then(([err, res]) => cb(err, res));
+export function verifySignatures(hmacKey, msgs, cb) {
+  wrapped.verifySignatures(hmacKey, msgs).then(([err, res]) => cb(err, res));
```

Comment:
Now that we know that errors have been strings, we should update this part to be

```diff
-.then(([err, res]) => cb(err, res));
+.then(([err, res]) => cb(new Error(err), res));
```

or even better:

```js
.then(([err, res]) => err ? cb(new Error(err)) : cb(null, res));
```

Or even better is to create a function

```js
function convertResults(cb) {
  return ([err, res]) => err ? cb(new Error(err)) : cb(null, res);
}
```

And then utilize it as

```js
  wrapped.verifySignatures(hmacKey, msgs).then(convertResults(cb));
```